File: IMS/src/app/features/customer/dtos/customerPoliciesResponse.dto.ts
export interface CustomerPoliciesResponseDto {
  policyId: number;
  availablePolicyName: string;
  issuedDate: string; // Typically represented as an ISO 8601 string in TypeScript
  expiryDate: string; // Typically represented as an ISO 8601 string in TypeScript
  agentName: string;
  agentContact: string;
  BasePremium: number; // Assuming basePremium is a number, adjust type as necessary
}

File: IMSIntrim/Applications/DTOs/CustomerPoliciesResponseDto.cs
namespace IMSIntrim.Applications.DTOs
{
    public class CustomerPoliciesResponseDto 
    {
        public int PolicyId { get; set; }
        
        public required string AvailablePolicyName { get; set; }
        public required string AgentName { get; set; }
        public DateTime IssuedDate { get; set; } 
        public DateTime ExpiryDate { get; set; }
        public required decimal PremiumAmount { get; set; }
        //public int AgentId { get; set; } 
        public required string AgentContact { get; set; } 
    }
}
